### PR TITLE
Fix linters errors

### PR DESCRIPTION
**Pull request by Pyonguk**
### Fix linters errors

Fix HTML errors
Fix CSS errors

### CODE REVIEW

**Review by Pyonguk — COMMENTED** [review]
### Hey @Pyonguk,

This is a great project you have going on, well-done pal.

However, there are a few sections to improve before your project is approved. 🔴

**Required Changes** ♻️

Check the inline comments under the review for sections to improve.

**Review by Pyonguk — COMMENTED** [review]
_Apparently, you can't delete comments, so to leave only one comment I moved all the information to the first one._

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Pyonguk , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [x] You don't need to use the "Review changes" feature from GitHub twice. It is used as a review summary field with review status. Kindly delete one of the two "Review changes" fields you have used. 
- [x] In this project you are highly recommended to use Github inline comments as it lets the student which particular line or block of code to look for. Kindly make at least two inline comments. If you are not sure how, please follow this link and read the material. [the lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -32,49 +27,36 @@
...
+                    <button class="not-button" type="button"><ion-icon name="notifications" class="ion-icon"></ion-icon></button>
                 </li>
                 <li>
-                    <button class="finger-button"><ion-icon name="finger-print"></ion-icon></button>
+                    <button class="finger-button" type="button"><ion-icon name="finger-print" class="ion-icon"></ion-icon></button>
                 </li>
             </ul>
         </header>
 
         <div id="columns">
```
Comment (Pyonguk) [inline index.html:57]: - [ ] I recommend you stick to using semantic HTML at all times. Change the div id="columns" to a section tag.

File: style.css [hunk 2]
```diff
@@ -1,238 +1,248 @@
...
-    line-height: 0;
-    cursor: pointer;
-    vertical-align: middle;
+  margin: 0 10px 0 0;
+  padding: 8px;
+  border: 0;
+  background: transparent;
+}
+
+.end .cam-button,
```
Comment (Pyonguk) [inline style.css:104]: - [ ] There is no need to create so many classes, if they are not necessary and the only changes you are going to make our group and not individual, I recommend that you use a single class for everyone.

File: index.html [hunk 3]
```diff
@@ -32,49 +27,36 @@
...
             <main class="primary">
-                <iframe src="https://www.youtube.com/embed/ChuSddfohxo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
+                <iframe src="https://www.youtube.com/embed/ChuSddfohxo" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```
Comment (Pyonguk) [inline index.html:59]: - [ ] I recommend you stick to using semantic HTML at all times. Change the div id="columns" to a section tag.

File: style.css [hunk 4]
```diff
@@ -1,238 +1,248 @@
...
+.end .app-button,
+.end .not-button,
+.end .finger-button {
+  color: white;
```
Comment (Pyonguk) [inline style.css:108]: - [ ] There is no need to create so many classes, if they are not necessary and the only changes you are going to make our group and not individual, I recommend that you use a single class for everyone.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Pyonguk 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**